cmd/cli/job/describe.go

```diff
@@ -102,7 +102,7 @@ func (o *DescribeOptions) printHeaderData(cmd *cobra.Command, job *models.Job) {
 		{Left: "Name", Right: job.Name},
 		{Left: "Namespace", Right: job.Namespace},
 		{Left: "Type", Right: job.Type},
-		{Left: "State", Right: job.State.StateType},
+		{Left: "Liveness", Right: job.State.StateType},
```

Comment:
This is the state of the job, rather than the node, is Liveness the correct word here?

Reply:
whoops, my editor was a little overzealous with the name change, will revert this.